modules/installation-osp-local-disk-deployment.adoc

```diff
@@ -0,0 +1,295 @@
++
+.Example YAML file
+[source,yaml]
+----
+# ...
+controlPlane:
+  name: master
+  platform:
+    openstack:
+      type: ${CONTROL_PLANE_FLAVOR}
+      rootVolume:
+        size: 25
+        types:
+        - ${CINDER_TYPE}
+  replicas: 3
+# ...
+----
+
+. Deploy the cluster you created by running the following command:
++
+[source,terminal]
+----
+$ openshift-install create cluster --dir <installation_directory> <1>
+----
++
+<1> For `<installation_directory>`, specify the location of the customized `./install-config.yaml file` that you previously created.
++
+
+. Verify that the cluster you deployed is healthy before proceeding to the next step by running the following command:
++
+[source,terminal]
+----
+$ oc wait clusteroperators --all --for=condition=Progressing=false <1>
+----
++
+<1> Ensures that the cluster operators are finished progressing and that the cluster is not deploying or updating.
+
+. Edit the `ControlPlaneMachineSet` (CPMS) to add the additional block ephemeral device that is used by etcd by running the following command:
++
+[%collapsible]
+====
+[source,terminal]
+----
+$ oc patch ControlPlaneMachineSet/cluster -n openshift-machine-api --type json -p ' <1>
+[
+    {
+      "op": "add",
+      "path": "/spec/template/machines_v1beta1_machine_openshift_io/spec/providerSpec/value/additionalBlockDevices", <2>
+      "value": [
+        {
+          "name": "etcd",
+          "sizeGiB": 10,
+          "storage": {
+            "type": "Local" <3>
+          }
+        }
+      ]
+    }
+  ]
+'
+----
+<1> Applies the JSON patch to the `ControlPlaneMachineSet` custom resource (CR).
+<2> Specifies the path where the `additionalBlockDevices` are added.
+<3> Adds the etcd devices with at least local storage of 10 GB to the cluster. You can specify values greater than 10 GB as long as the etcd device fits the Nova flavor. For example, if the Nova flavor has 15 GB, you can create the etcd device with 12 GB.
+====
+
+. Verify that the control plane machines are healthy by using the following steps:
+
+.. Wait for the control plane machine set update to finish by running the following command:
+
++
+[source,terminal]
+----
+$ oc wait --timeout=90m --for=condition=Progressing=false controlplanemachineset.machine.openshift.io -n openshift-machine-api cluster
+----
+
+.. Verify that the 3 control plane machine sets are updated by running the following command:
+
++
+[source,terminal]
+----
+$ oc wait --timeout=90m --for=jsonpath='{.status.updatedReplicas}'=3 controlplanemachineset.machine.openshift.io -n openshift-machine-api cluster
+----
+
+.. Verify that the 3 control plane machine sets are healthy by running the following command:
+
++
+[source,terminal]
+----
+$ oc wait --timeout=90m --for=jsonpath='{.status.replicas}'=3 controlplanemachineset.machine.openshift.io -n openshift-machine-api cluster
+----
+
+.. Verify that the `ClusterOperators` are progressing in the cluster by running the following command:
```

Comment:
are _not_ progressing, right?

Reply:
Yeah, this is a weird one because you want to make sure that they are progressing by verifying that they are not progressing.